entrypoint.sh

```diff
@@ -241,7 +241,8 @@ configure_nvidia_installation_dirs() {
   update_container_ld_cache
 
   # Install an exit handler to cleanup the overlayfs mount points.
-  trap "{ umount /usr/lib/x86_64-linux-gnu ; umount /usr/bin; }" EXIT
+  trap "{ umount /lib/modules/$(uname -r)/video; umount /usr/lib/x86_64-linux-gnu ; umount /usr/bin; }" EXIT
```

Comment:
thanks for catching the missing umount path!  please add quotes around `uname`:

`trap "{ umount /lib/modules/\"$(uname -r)\"/video; umount /usr/lib/x86_64-linux-gnu ; umount /usr/bin; }"`

Reply:
Done.